specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/trafficmanager.json

```diff
@@ -0,0 +1,1578 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "TrafficManagerManagementClient",
+    "version": "2018-03-01"
```

Comment:
folder name is 2018-04-01 but the version here is still 2018-03-01. Do you intend to add the APIs in a newer APi-version or existing?

Reply:
That was in error. Fixed in https://github.com/Azure/azure-rest-api-specs/pull/3360